sdk/include/cardboard.h

```diff
@@ -433,6 +433,29 @@ void CardboardQrCode_getSavedDeviceParams(uint8_t** encoded_device_params,
 ///     using cardboard_device.proto.
 void CardboardQrCode_destroy(const uint8_t* encoded_device_params);
 
+/// Sets and saves the encoded device parameters provided by the uri.
+///
+/// @details This function gets the device parameters from the URI passed as a
+///          parameter. Then sets and saves the encoded device parameters
+///          provided by the uri.
+///          Expected URI format for:
+///          cardboard V1: https://g.co/cardboard
+///          cardboard V2: https://google.com/cardboard/cfd?p=<deviceParams>
+///          URI example for cardboard v2:
+///          "https://google.com/cardboard/cfg?p=CgZHb29nbGUSEkNhcmRib2FyZCBJL
+///          08gMjAxNR0rGBU9JQHegj0qEAAASEIAAEhCAABIQgAASEJYADUpXA89OggeZnc-Ej
+///          6aPlAAYAM"
+///          Redirection is also supported up to a maximum of 5 possible
+///          redirects to reach the proper pattern.
+///
+/// @pre @p uri Must not be null.
+/// @pre @p size Must be greater than 0.
+///
+/// @param[in]      uri                     URI with parameters of the device
+/// @param[in]      size                    is the size of @p uri
+///
+void CardboardQrCode_setAndSaveDeviceParams(const uint8_t* uri, int size);
```

Comment:
Per comment on PR feed, consider naming this "CardboardQrCode_saveDeviceParams". Please fix here and there.